config/locales/client.en.yml

```
@@ -637,7 +637,7 @@ en:
       private_message: "Message"
       private_messages: "Messages"
       ignore: "Ignore"
-      watch: "Watch"
+      unignore: "Un-ignore"
```

```suggestion
      unignore: "Unignore"
```

I don't believe the hyphen is necessary here.